test/correctness/strict_float.cpp

```diff
@@ -257,6 +257,11 @@ Buffer<float> block_transposed_by_n(Buffer<float> &buf, int vectorize) {
 }
 
 int main(int argc, char **argv) {
+    if (get_jit_target_from_environment() != get_host_target()) {
```

Comment:
Are you sure about this one? It looks like it tests strict_float on a variety of sse/avx levels where llvm may do different types of fusion.